lib/LMSManagers/LMSEventManager.php

```diff
@@ -270,7 +270,8 @@ public function GetEvent($id)
         }
 
         $event['helpdesk'] = !empty($event['ticketid']);
-        $event['userlist'] = $this->db->GetCol('SELECT userid AS id
+
+        $event['userlist'] = $this->db->GetCol('SELECT rname, userid AS id
```

Comment:
GetCol() dotyczy pobrania jednej kolumy - to nie będzie działać.

Reply:
Akurat tutaj w UI nie widziałem problemu ale niech będzie prawilnie 857122874b46fd93ef772e0647e8e64806911e87.

Reply:
> Akurat tutaj w UI nie widziałem problemu ale niech będzie prawilnie [8571228](https://github.com/chilek/lms/commit/857122874b46fd93ef772e0647e8e64806911e87).

WIdocznie słabo testowałeś, bo z GetCol() nie mogło to sensownie działać.